extensions/amp-analytics/0.1/vendors.js

```diff
@@ -192,6 +192,34 @@ export const ANALYTICS_CONFIG = /** @type {!JsonObject} */ ({
     },
   },
 
+  'umenganalytics': {
+    'vars': {
+      'siteid': '',
+      'initial_view_time': '',
+      'eventName': '',
+      'eventProps': '',
+    },
+    'requests': {
```

Comment:
Just to confirm that you don't provide default trigger in your vendor config? even `pageview` event?

Reply:
Yes I confirm，Because if there are multiple amp-analytics components with the same type on a page, the default pageview will be trigger multiple times. So we removed it. And let the users decide whether to use it.

Reply:
That sounds good to me as long as you document the usage clearly on your site : )